documentation/docs/03_account.md

```diff
@@ -1,2 +1,218 @@
+# Account Interface
+
 The Account Interface is evaluated, after the Account Manager Interface, repeatedly through a prompt within the `wallet`
 binary.
+
+It is responsible for the creation and management of account addresses and their outputs, tokens, native tokens, NFTs...
+
+## Commands
+
+### `addresses`
+
+#### Parameters
+
+| Name    | Optional  | Default       | Example |
+| ------- | --------- | ------------- | ------- |
+| `` | ✓ | | |
+
+#### Example(s)
+
+```sh
+> Account "main": addresses
+```
+
+### `balance`
+
+#### Parameters
+
+| Name    | Optional  | Default       | Example |
+| ------- | --------- | ------------- | ------- |
+| `` | ✓ | | |
+
+#### Example(s)
+
+```sh
+> Account "main": balance
+```
+
+### `consolidate`
+
+#### Parameters
+
+| Name    | Optional  | Default       | Example |
+| ------- | --------- | ------------- | ------- |
+| `` | ✓ | | |
+
+#### Example(s)
+
+```sh
+> Account "main": consolidate
+```
+
+### `exit`
+
+#### Parameters
+
+| Name    | Optional  | Default       | Example |
+| ------- | --------- | ------------- | ------- |
+| `` | ✓ | | |
+
+#### Example(s)
+
+```sh
+> Account "main": exit
+```
+
+### `faucet`
+
+#### Parameters
+
+| Name    | Optional  | Default       | Example |
+| ------- | --------- | ------------- | ------- |
+| `` | ✓ | | |
```

Comment:
There are optonal url and address parameters

Reply:
Yeah I'm doing this page today, just ignore for now